tests/remote-cache/Earthfile

```diff
@@ -1,9 +1,28 @@
-VERSION 0.6
-ARG DOCKERHUB_USER_SECRET=+secrets/DOCKERHUB_USER
-ARG DOCKERHUB_TOKEN_SECRET=+secrets/DOCKERHUB_TOKEN
-ARG DOCKERHUB_MIRROR
-ARG DOCKERHUB_MIRROR_INSECURE=false
-ARG DOCKERHUB_AUTH=true
+VERSION                             \
+--check-duplicate-images            \
+--ci-arg                            \
+--earthly-git-author-args           \
+--earthly-locally-arg               \
+--earthly-version-arg               \
+--explicit-global                   \
+--git-commit-author-timestamp       \
+--new-platform                      \
+--no-tar-build-output               \
+--shell-out-anywhere                \
+--use-cache-command                 \
+--use-chmod                         \
+--use-copy-link                     \
+--use-host-command                  \
+--use-no-manifest-list              \
+--use-pipelines                     \
+--use-project-secrets               \
+#--wait-block                        \
```

Comment:
Was this comment intended to be left in?

Reply:
Oh, I think this is the bug that we're fixing after the RC, right? got it.

Reply:
This whole list comes from https://github.com/earthly/earthly/issues/2237#issuecomment-1340120452 (the features we're enabling for v0.7); I'm inclined to leave it commented out to both illustrate it's been disabled on purpose -- and to make it easier to re-enable when we loop back to fix this after the initial RC.

Reply:
Yeah, makes sense. I usually use `git revert` for those purposes, so I don't often leave the comments in - but it does serve as a good reminder.